docs/resources/rego_policy_instance.md

```diff
@@ -71,167 +71,19 @@ All templates use parameters defined as JSON, below is a list of all the corresp
--   `labels` (Array) Data Labels.
--   `identities` (Object) Identities associated to the policy. If empty, the policy will be associated to all identities. See [identities](#objects--identities).
--   `dbAccounts` (Object) Database Accounts associated to the policy. If empty, the policy will be associated to any database account. See [dbAccounts](#objects--dbAccounts).
--   `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
-
-### Ephemeral Grant (EphemeralGrantPolicy)
-
--   `repoAccount` (String) Repository Account Name.
--   `repo` (String) Repository Name.
--   `allowedSensitiveAttributes` (Array) Allowed Sensitive Attributes.
-
-### Rate Limit (rate-limit)
-
--   `rateLimit` (Integer) Maximum number of rows that can be returned per hour. Note: the value must be an integer greater than zero.
--   `block` (Boolean) Policy action to enforce.
--   `tags` (Array) Tags.
--   `labels` (Array) Data Labels.
--   `identities` (Object) Identities associated to the policy. If empty, the policy will be associated to all identities. See [identities](#objects--identities).
--   `dbAccounts` (Object) Database Accounts associated to the policy. If empty, the policy will be associated to any database account. See [dbAccounts](#objects--dbAccounts).
--   `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
-
-### Read Limit (read-limit)
-
--   `rowLimit` (Integer) Maximum number of rows that can be read per query. Note: the value must be an integer greater than zero.
--   `block` (Boolean) Policy action to enforce.
--   `appliesToAllData` (Boolean) Whether the policy should apply to the entire repository data.
--   `tags` (Array) Tags.
--   `labels` (Array) Data Labels.
--   `identities` (Object) Identities associated to the policy. If empty, the policy will be associated to all identities. See [identities](#objects--identities).
--   `dbAccounts` (Object) Database Accounts associated to the policy. If empty, the policy will be associated to any database account. See [dbAccounts](#objects--dbAccounts).
--   `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
-
-### Repository Protection (repository-protection)
+### Object Protection (object-protection)
 
--   `rowLimit` (Integer) Maximum number of rows that can be modified per query. Note: the value must be an integer greater than zero.
--   `monitorUpdates` (Boolean) Monitor update operations.
--   `monitorDeletes` (Boolean) Monitor delete operations.
--   `identities` (Object) Identities associated to the policy. If empty, the policy will be associated to all identities. See [identities](#objects--identities).
--   `dbAccounts` (Object) Database Accounts associated to the policy. If empty, the policy will be associated to any database account. See [dbAccounts](#objects--dbAccounts).
```

Comment:
Is it possible to retain the reference to #object--identities and #objects--dbAccounts in the description below?